vNext/test/Tester/Tester.csproj

```diff
@@ -57,6 +57,39 @@
       <Project>{40ee3b00-d381-485f-9c69-ff706837deed}</Project>
       <Name>OrleansTestingHost</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\src\Orleans\Orleans.csproj">
```

Comment:
ah, looking further into #2323 I still see that it uses the Tester project instead of a new test project... oh well, I guess we'll have to postpone that cleanup instead of doing it at this same time 😢